### ISSUE TITLE
Use NODERAWFS during build to create the boot file

### ISSUE THREAD

**Issue by rla** [SWI-Prolog/swipl-wasm#3]
It seems like the seek issue is fixed but yet not released (not in Emscripten 1.38.8).
Requires modification to `src/Makefile.in`.

Reference: <https://github.com/kripken/emscripten/issues/6612>

**Comment by JanWielemaker** [comment]
Great!  At least being able to use nodejs simplifies building and scripted testing.  Otherwise I think I'd only advocate this with many disclaimers.  After all, the wasm version is fairly crippled when compared to the real thing (no threads, limited access to the OS, 32-bit limits the stack space, etc.)

**Comment by rla** [comment]
Yes, the idea is to make wasm usage as simple as possible without making other things too complex (current Makefile) or breaking them. I'm currently waiting until the fix gets into a Emscripten release (it should be the next release) and then I start working on things and I definitely want to use automated tests.
